plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -257,11 +262,11 @@ def check_link_down_condition(self, port_obj, port_name, ports_counters):
         #TODO check for a way to save peer row in data structure for performance
         peer_row = ports_counters.loc[(ports_counters['port_guid'] == peer_guid) & (ports_counters['port_num'] == int(peer_num))]
         if peer_row.empty:
-            self.logger.warning("Peer port {0} not found in ports data".format(peer_port_name))
+            self.logger.warning("Peer port {0} not found in ports data".format(port_obj.peer))
```

Comment:
port_obj.peer - ist it an object? what will be the string value for port_obj.peer???

Reply:
4 row above you can see it is string value, as it uses split